go/private/context.bzl

```diff
@@ -313,16 +313,14 @@ def _library_to_source(go, attr, library, coverage_instrumented, verify_resolver
 
         # TODO(zbarsky): Remove this once downstream has a chance to migrate.
         if verify_resolver_deps:
-            has_targets = False
             for dep in source["deps"]:
                 if type(dep) == "Target":
-                    has_targets = True
+                    print('Detected Targets in `source["deps"]` as a result of _resolver: ' +
```

Comment:
since we break, we only ever use `has_targets` once, so there's no need to pull that branch out of the loop.

Reply:
yep, nice

Reply:
I wasn't sure if replacing the list while iterating would be an issue, but I suppose we're just replacing the reference so it's fine